web/src/components/notification-pill/notification-pill.css

```diff
@@ -50,3 +50,15 @@
         }
     }
 }
+
+.achievement {
+    line-height: 1.5;
+    img {
+        width: 14px;
+        height: 14px;
+        margin-right: 10px;
+    }
+    .score {
+        display: inline-block;
+    }
+}
```

Comment:
Is this the one that appears outside of the dashboard? Any way we can make the selector a bit more specific?